internal/commands/scan.go

```diff
@@ -384,6 +387,37 @@ func scanShowSubCommand(scansWrapper wrappers.ScansWrapper) *cobra.Command {
 	return showScanCmd
 }
 
+func scanVorpalSubCommand() *cobra.Command {
+	scanVorpalCmd := &cobra.Command{
+		Use:    "vorpal",
+		Hidden: true,
+		Short:  "Run a vorpal scan",
+		Long:   "vorpal scan is able to scan a single file fast and efficiently.",
+		Example: heredoc.Doc(
+			`
+			$ cx scan vorpal --file-source <path to code file> --vorpal-latest-version <use latest Vorpal available version>
+		`,
+		),
+		Annotations: map[string]string{
+			"command:doc": heredoc.Doc(
+				`
+				https://checkmarx.com/resource/documents/en/link-to-cxone-documentation
```

Comment:
link-to-cxone-documentation ? what is mean? this is how it's in another command?

Reply:
It's a placeholder, which is hidden, until Eli uploads the documentation and then I'll be able to use the link.